lib/ts/recipe/thirdpartypasswordless/recipe.tsx

```diff
@@ -56,61 +62,48 @@ export default class ThirdPartyPasswordless extends AuthRecipe<
 
     thirdPartyRecipe: ThirdParty | undefined;
 
-    recipeImpl: TPPWlessRecipeInterface;
+    recipeImpl: WebJSRecipeInterface;
 
     constructor(
-        config: Config,
+        config: NormalisedConfigWithAppInfoAndRecipeID<NormalisedConfig>,
         recipes: {
             thirdPartyInstance: ThirdParty | undefined;
             passwordlessInstance: Passwordless | undefined;
         }
     ) {
-        super(normaliseThirdPartyPasswordlessConfig(config));
-
-        {
-            const builder = new OverrideableBuilder(
-                RecipeImplementation({
-                    appInfo: this.config.appInfo,
-                    recipeId: this.config.recipeId,
-                    onHandleEvent: this.config.onHandleEvent,
-                    preAPIHook: this.config.preAPIHook,
-                    postAPIHook: this.config.postAPIHook,
-                })
-            );
-            this.recipeImpl = builder.override(this.config.override.functions).build();
-        }
+        super(config);
+
+        this.recipeImpl = ThirdpartyPasswordlessWebJS;
 
         this.passwordlessRecipe =
             recipes.passwordlessInstance !== undefined
                 ? recipes.passwordlessInstance
-                : this.config.passwordlessUserInput === undefined
+                : this.config.passwordlessConfig === undefined
                 ? undefined
-                : new Passwordless({
-                      ...this.config.passwordlessUserInput,
-                      appInfo: this.config.appInfo,
-                      recipeId: this.config.recipeId,
-                      override: {
-                          ...this.config.passwordlessUserInput.override,
-                          functions: () => getPasswordlessImpl(this.recipeImpl),
+                : new Passwordless(
+                      {
+                          ...this.config.passwordlessConfig,
+                          appInfo: this.config.appInfo,
+                          recipeId: this.config.recipeId,
                       },
-                  });
+                      getPasswordlessImpl(this.recipeImpl)
```

Comment:
can we git rid of getPasswordlessImpl and getThirdPartyImpl functions too? Instead, pick them up from webJS SDK? What's happening now is also fine, but just would have been "cleaner" if we don't have these at all.

Reply:
i might be wrong but  i think recipeImpl here `getPasswordlessImpl(this.recipeImpl)` is coming from web-js `thirdpartypasswordless` and has sub-recipes initialized and if we were to import those directly separately it would throw an error saying instance of recipe is not found

Reply:
Why can’t we do something like this? `ThirdPartyPasswordless.getInstanceOrThrow().passwordless.recipeImpl`